src/solana/rpc/async_api.py

```diff
@@ -574,7 +593,7 @@ async def get_multiple_accounts(
             >>> from solana.publickey import PublicKey
             >>> solana_client = AsyncClient("http://localhost:8899")
             >>> pubkeys = [PublicKey("6ZWcsUiWJ63awprYmbZgBQSreqYZ4s6opowP4b7boUdh"), PublicKey("HkcE9sqQAnjJtECiFsqGMNmUho3ptXkapUPAqgZQbBSY")]
-            >>> (await solana_client.get_multiple_accounts(pubkeys)).value[0].lamports # doctest: +SKIP
+            >>> (await solana_client.multiple_accounts(pubkeys)).value[0].lamports # doctest: +SKIP
```

Comment:
Guessing this is accidental, please undo

Reply:
:thinking: I don't have that on my local code. I've probably messed up with the commits. That was a mistake.